source/nodejs/adaptivecards-designer/src/open-sample-dialog.ts

```diff
@@ -74,8 +74,7 @@ class CatalogueItem {
                     card.renderedElement.style.width = "100%";
 
                     thumbnailHost.appendChild(card.renderedElement);
-                }
-                catch (e) {
+                } catch (e) {
                     // Swallow the exception
```

Comment:
Catch should be on a new line after }, as it was before

Reply:
@dclaux  Addressed the review comments.